data/ovs/001-ovs-cni.yaml

```diff
@@ -59,12 +55,60 @@ spec:
                 fieldRef:
                   fieldPath: spec.nodeName
       volumes:
-        - name: localbin
```

Comment:
@phoracek is this necessary?

Reply:
I think it is just a leftover that is not being used anymore